rust/core-lib/src/width_cache.rs

```diff
@@ -19,44 +19,93 @@ use std::collections::{BTreeMap, HashMap};
 
 use xi_rpc;
 
-use client::{Client, WidthReq};
+use client::Client;
 
 /// A token which can be used to retrieve an actual width value when the
 /// batch request is submitted.
 ///
 /// Internally, it is implemented as an index into the `widths` array.
 pub type Token = usize;
 
+/// A measured width, in points.
```

Comment:
I prefer "px units", "points" is macOS specific and misleading (it can easily be confused with 1/72 inch).